includes/WikiDiscover.php

```diff
@@ -134,6 +134,21 @@ public static function onParserGetVariableValueSwitch(
 			global $wgLocalDatabases;
 			$ret = $cache[$magicWordId] = count( $wgLocalDatabases );
 		}
+		if ( $magicWordId == 'numberofprivatewikis' ) {
+			global $wgCreateWikiDatabase;
+			$dbw = wfGetDB( DB_MASTER, [], $wgCreateWikiDatabase );
+			$ret = $cache['numberofprivatewikis'] = $dbw->selectRowCount( 'cw_wikis', '*', 'wiki_private = 1' );
```

Comment:
Why not $magicWordId?

Reply:
Same for below, why are we not following standard of [ $table => $value ]?

Reply:
> Why not $magicWordId?

Oh, sorry that was unintentional when I was testing it I tested multiple ways and forgot to revert it back, I will fix that, thank you!